src/call_builder.js

```diff
@@ -1,14 +1,13 @@
 import forEach from 'lodash/forEach';
 import URI from 'urijs';
 import URITemplate from 'urijs/src/URITemplate';
+import { NativeEventSource, EventSourcePolyfill } from 'event-source-polyfill';
```

Comment:
AFAIK it's generally best practice to ask consumers to provide polyfills for web standards rather than pulling them in as dependencies. That would be a breaking change so I don't think we should make it willy-nilly, but at some point it might be better to rely on `window.EventSource` and just provide a warning if it's missing.